### PR TITLE
added xarm robot

### PR DESCRIPTION
**Pull request by DanielWang123321**
added xarm robot
(2020/07/29)

### CODE REVIEW

File: _posts/2020-07-29-xarm.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- 'manipulator'
+- 'robotic arm'
+- 'ufactory'
+- 'xarm'
+- 'xarm-robots'
+twitter_text: Checkout the xArm
+website: https://xarm.cc/
+wiki_homepage: http://wiki.ros.org/xarm
+---
+
```
Comment (tfoote) [inline _posts/2020-07-29-xarm.md:24]: Please add at least a sentence or two overview.

File: .idea/inspectionProfiles/Project_Default.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<component name="InspectionProjectProfileManager">
```
Comment (tfoote) [inline .idea/inspectionProfiles/Project_Default.xml]: This and all the other .idea folder files should be removed from the PR

**Review by tfoote — CHANGES_REQUESTED** [review]

File: _posts/2020-07-29-xarm.md (hunk 1 above)
Comment (DanielWang123321) [reply _posts/2020-07-29-xarm.md:24]: ok, done. thanks.

**Review by DanielWang123321 — COMMENTED** [review]
modified, thanks.

File: .idea/inspectionProfiles/Project_Default.xml (hunk 2 above)
Comment (DanielWang123321) [reply .idea/inspectionProfiles/Project_Default.xml]: deleted

**Review by tfoote — APPROVED** [review]

**Pull request merged**